csrc/moe/torch_bindings.cpp

```diff
@@ -1,12 +1,29 @@
 #include "registration.h"
 #include "moe_ops.h"
+#include "marlin_moe_ops.h"
+
+#include <torch/library.h>
 
 TORCH_LIBRARY_EXPAND(TORCH_EXTENSION_NAME, m) {
   // Apply topk softmax to the gating outputs.
   m.def(
       "topk_softmax(Tensor! topk_weights, Tensor! topk_indices, Tensor! "
       "token_expert_indices, Tensor gating_output) -> ()");
   m.impl("topk_softmax", torch::kCUDA, &topk_softmax);
+
+  m.def("marlin_gemm_moe(Tensor! a, Tensor! b_q_weights, Tensor! sorted_ids, Tensor! topk_weights, "
+                        "Tensor! b_scales, Tensor! expert_offsets, Tensor! workspace, size_m, size_n, size_k, "
+                        "num_tokens_post_padded, num_experts, topk, moe_block_size, replicate_input, apply_weights) -> Tensor");
```

Comment:
all the ints here will need `int` type, e.g. `int size_m, int size_n, int size_k` etc.